### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by luigifab** [OpenMage/magento-lts#2282] [General]
There are many PR added for next 19.x and 20.x releases.
I like that, but this is dangerous.

We are not Gnome developers that from 3.4.5.6 to 3.4.5.7 they drop all icons in menu and says: you dislike? we prefer like that! your application crash? update your code.

What are the next release numbers? and when start the (crash) test period?
(I use many PR not merged, and this is time consuming to apply them to test the latest commit)

**Comment by addison74** [comment]
I proposed the same issue for discussion. What shall we do in the near future? Wait for new archived releases as before or from now on we have to clone the repository.

For new releases we should decide when this action should be done, by PRs number, every 3 months. Also, the new releases should be renamed. I consider that it no longer works with 1.9.4.X and 20.X. We have two variants of OpenMage and there should be a continuity in numbering as simple as possible identify the version when there are reports of problems.

The 20.x branch should be the default one and we should evaluate if we go ahead with one or two versions. It is too much effort to manage two options.

**Reply by Flyingmana** [reply]
i will try to get this rfc ready this month https://github.com/OpenMage/rfcs/blob/e18f1a53d9633bc65141337dad5ceee2793759d9/draft/0000-release-schedule.md
so we have clarity n versions and related plans.

but sounds like you want to add a possible alternative to it

**Comment by fballiano** [comment]
on 19th july there's a meetup on discord, let's get together and talk about it then, i think it's much easier

**Reply by luigifab** [reply]
I suppose this link https://discord.gg/EV8aNbU ?
I will think to create an account.

At Fedora mailing lists, there is a command to convert the meeting date in your local time zone: `date -d '2022-07-19 20:00 UTC'`

![image](https://user-images.githubusercontent.com/31816829/179011248-4226d3e7-1325-44f5-a4f2-c5e6adb854ea.png)
